src/platforms/apple/common/install/swift-package-manager.mdx

```diff
@@ -10,6 +10,16 @@ https://github.com/getsentry/sentry-cocoa.git
 ```
 
 You can define your dependency rule by selecting the SDK version (or branch), and then click the "Add Package" button.
+You will then be prompt to choose between 3 options `Sentry`, `Sentry-Dynamic` and `SentrySwiftUI`.
+
+- `Sentry` is the static framework, which is the recommended option if you prefer a fast app start time.
+- `Sentry-Dynamic` is the dynamic framework, which is the recommended options if you more control.
```

Comment:
```suggestion
- `Sentry-Dynamic` is the dynamic framework, which is the recommended option if you require more control.
```

Reply:
what kind of control?

Reply:
Good question. It did more sense in my mind.